app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,49 @@
+class QuestionsController < ApplicationController
+  before_action :ensure_logged_in, only:[:new, :create, :update]
+
+  def index
+    @questions = Question.all.order(created_at: :desc)
```

Comment:
You could use a paginator and a limit clause
`Question.all.order(created_at: :desc).limit(20)` for example